src/components/PatchesPage/index.tsx

```diff
@@ -167,3 +193,8 @@ const CheckboxContainer = styled.div`
   display: flex;
   justify-content: end;
 `;
+
+/* @ts-expect-error */
+const StyledCheckbox = styled(Checkbox)`
+  padding-left: ${size.xs};
+`;
```

Comment:
Similar note as above, you can add a `gap` to `CheckboxContainer` instead of creating a specialized checkbox.